concordia/static/js/base.js

```diff
@@ -68,6 +68,44 @@ function displayMessage(level, message, uniqueId) {
     $messages.append($newMessage);
 }
 
+function displayHTMLMessage(level, message, uniqueId) {
+    /*
+        Display an HTML dismissable message at a level which will match one of the
+        Bootstrap alert classes
+        (https://getbootstrap.com/docs/4.1/components/alerts/)
+
+        If provided, uniqueId will be used to remove any existing elements which
+        have that ID, allowing old messages to be replaced automatically.
+    */
+    var $messages = $('#messages');
+    $messages.removeAttr('hidden');
+
+    var $newMessage = $messages
+        .find('#message-template .alert')
+        .clone()
+        .removeAttr('hidden')
+        .removeAttr('id');
+
+    $newMessage.addClass('alert-' + level);
+
+    if (uniqueId) {
+        $('#' + uniqueId).remove();
+        $newMessage.attr('id', uniqueId);
+    }
+
+    $newMessage.prepend(message);
+
+    $messages.append($newMessage);
+}
+
+function displayMessage(level, message, uniqueId, isText = true) {
```

Comment:
Unfortunately, this won't work in IE since it doesn't support ES5:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/default_parameters